app/views/products/index.html.erb

```diff
@@ -17,7 +17,7 @@
         <% end %>
         <td><%= product.name %></td>
         <td><%= product.description %></td>
-        <td><%= number_to_currency product.unit_price %></td>
+        <td><%= number_to_currency product&.unit_price_in_cents %></td>
```

Comment:
Number to currency expects a dollars amount, should this just a lonely operator vs the change to `unit_price_in_cents`